### ISSUE TITLE
[CAIA A11y Research]: VA BTSSS (Travel Pay), Travel Pay on VA.gov (EOM April 2024)

### ISSUE THREAD

**Issue by carafrissell1** [department-of-veterans-affairs/va.gov-team#80740]
> [!NOTE]
> **High-Level Overview**: `VA BTSSS (Travel Pay)`
> - **Initial Kickoff**: `Date and Time TBD`
> - **Research Timeframe**: `4/19-5/1/2024`
> - **Time of Day:** `Afternoon`
> - **Pilot**:  `4/18/2024` #81037 81037
> - **Assistive Tech Users**: `3` AT Users
> - **Artifacts Available**: `TBD`
> - **Testing Environment:** `CodePen Prototype`
> - **Assistive Tech Being Used**: `Screen Readers` `Types - TBD`
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- @coforma-jamie 
- @EvanAtCoforma 
- @SarahKay8 

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- Team name: VA BTSSS (Travel Pay)
- OCTO-DE product owner: Kay Lawyer 
- Product manager: Ayush Chakravarty 
- Designer: Micah Taylor 
- FE engineer: Kevin Duensing 
- Product/team Slack channel:#beneficiary-travel-team

</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Cara Frissell, Micah Taylor

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe
</details>

## Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

- **Pilot Session**:`4/18/2024`at noon
- **Timeframe**: `4`/`19`/2024 - `5`/`1`/2024

### Length of sessions

- **Session length:** `60` minutes
- **Buffer time between sessions:** `pending` minutes
- **Maximum sessions per day:** `pending`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
P5 `tech issues, need to reschedule` | Friday, April 19, 11am - noon EST | Magnilink | Desktop; Windows | Chrome | `TBD` | @coforma-jamie  
P15 | Friday, April 26, 12:30 - 1:30pm EST | VoiceOver | Desktop; MacOS | Beginner | `TBD` | @EvanAtCoforma   
P16 | Tuesday, April 30, 1:30 - 3pm EST | unknown | unknown | Beginner | `TBD` | @SarahKay8  
P17 | Wednesday, May 1, 11am - noon EST | Narrator | Desktop; Windows 11 | Beginner | `TBD` | @coforma-jamie  

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/beneficiary-travel)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1708602401825/9786edc7ee2a5b6c4efd1a895d067a07ae2a7f27?wid=0-1709668560371&sender=ub0f383fafad4c2df7fc53935)
- [Prototype or mockups](https://codepen.io/team/VA-gov-Applications/project/live/DQmqxR)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/beneficiary-travel/research/2024-04-Reimbursement-Status-Look-and-Feel-Study/ResearchPlan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/beneficiary-travel/research/2024-04-Reimbursement-Status-Look-and-Feel-Study/ConversationGuide.md)

</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket
- [x] **INITIAL MEETING:** Schedule meeting with team
- [x] **(OPTIONAL) FEEDBACK REQUESTED:** Review Conversation Guide @coforma-jamie
- [x] **PILOT SESSION: SCHEDULE** Pilot session with team. 4/18/2024
- [x] **PILOT SESSION: CREATE [TICKET](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Csara-amanda&labels=Sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&projects=&template=CAIA-internal-Research-Pilot-Ticket.md&title=Internal+Pilot+Research+Ticket)** Link ticket in this task list. [81037](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81037)
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available. @sara-amanda
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean) @coforma-jamie @sara-amanda
- [x] **RESEARCH NOTES** Create CAIA notes document @coforma-jamie
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support. @coforma-jamie @EvanAtCoforma
- [x] **Review research folder and assets**  @coforma-jamie @EvanAtCoforma
- [ ] **Participate in research**  **Date Span:**`4/19/2024` and `5/1/2024` - `INVITES RCVD 4/17`
- [x] **SESSION NOTES**: Create [CAIA notes document](https://docs.google.com/document/d/12QYin6ym9JKiBdkDC-4KK56oiKvknLDduzIXImFJBoA/edit) for a11ys to use during the sessions
- [ ] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [ ] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [ ] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/81037
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @coforma-terry in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH) @carafrissell1 [4/15/2024 via Slack](https://dsva.slack.com/archives/C8E985R32/p1713375346100459?thread_ts=1713203101.987449&cid=C8E985R32)
- [ ] Schedule intial meeting with @coforma-terry and @sara-amanda
- [x] Schedule a pilot session with CAIA a11ys (recommended) - time limits may prevent this for this study. 4/18/2024 at noon
- [x] (OPTIONAL) Receive feedback on Conversation Guide for Screen Readers (if requested)
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section
- [x] **DATES NEEDED** See `Toggle to view the AT testing schedule` in the ticket above. Please provide date range and slots for AT sessions being planned.
- [ ] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in the `Table` in the ticket above.
- [ ] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara or Terry)
```

**Comment by sara-amanda** [comment]
# CAIA Research Intake Processed

Hi, @carafrissell1! Just a quick note here to let you know that we received your intake ticket and it has been processed. Thank you for the invites, they have been added to our [CAIA A11y Capacity calendar](https://calendar.google.com/calendar/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a%40group.calendar.google.com&ctz=America%2FNew_York).

@coforma-jamie and @EvanAtCoforma will be assisting on this study.  

## Next Steps
- [ ] Provide details in the participant table of this ticket about the assistive technology, browser, device and the self-expressed skill level of the user.
- [ ] If you can, as Jamie mentioned in [Slack](https://dsva.slack.com/archives/C8E985R32/p1713203101987449), avoid the users logging in, that would be ideal to avoid any blockers, for users of screen readers. 
- [ ] We typically recommend a pilot, prior to testing, but since testing is coming up so quickly, we realize this may not be an option, at this time. 
- [ ] There are two task lists in this ticket: one for CAIA and one for your team to use, as we progress through the study. 
- [ ] Please feel free to reach out to us with any questions you may have. We are excited to help you! 

All the best,

Sara

cc: @coforma-terry

**Comment by carafrissell1** [comment]
Hi @sara-amanda @coforma-jamie @EvanAtCoforma @coforma-terry  we had another AT user sign up for April 30, 2024 at 1:30pm EST.  If I'm reading the calendar correctly, it looks like there's capacity for the first half.  Is anyone available to support this session? I'm tracking down the technical details needed.  Thanks for all your help!

**Comment by coforma-jamie** [comment]
I think @EvanAtCoforma and I have an internal conflict at 2p. @sara-amanda, are you free then?

**Comment by sara-amanda** [comment]
@carafrissell1 can you send me the invite for the 1:30 p.m. 4/30 session? I can add it to our capacity calendar. Thanks so much!

**Comment by carafrissell1** [comment]
@sara-amanda sent.  I also sent an invite to Sarah Koomson who I have as the support for that session.

**Comment by sara-amanda** [comment]
Thank you so much, @carafrissell1! Adding to our calendar, now. 👍

**Comment by carafrissell1** [comment]
Hey @sara-amanda @SarahKay8 @coforma-jamie we have decided to end the study early and next weeks' sessions are canceled.  We no longer need Accessibility support.  Thanks for your help and flexibility in this process.

**Comment by sara-amanda** [comment]
[Closing ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80740#issuecomment-2079936679) per Cara's comment above. Study is being suspended.
